auth/user_mgt.go

```diff
@@ -230,25 +256,26 @@ func (c *Client) Users(ctx context.Context, nextPageToken string) *UserIterator
 }
 
 func (it *UserIterator) fetch(pageSize int, pageToken string) (string, error) {
-	params := map[string]interface{}{"maxResults": pageSize}
-	if pageToken != "" {
-		params["nextPageToken"] = pageToken
+	request := &identitytoolkit.IdentitytoolkitRelyingpartyDownloadAccountRequest{
+		MaxResults:    int64(pageSize),
+		NextPageToken: pageToken,
 	}
-
-	var lur listUsersResponse
-	err := it.client.makeHTTPCall(it.ctx, "downloadAccount", params, &lur)
+	relyingpartyDownloadAccountCall := it.client.is.Relyingparty.DownloadAccount(request)
```

Comment:
Rename to `call`? Go advocates shorter names for local variables.

Reply:
perfect 👍